OpenRA.Mods.Common/Widgets/ProductionPaletteWidget.cs

```diff
@@ -484,7 +484,12 @@ public override void Draw()
 							icon.Pos + timeOffset,
 							Color.White, Color.Black, 1);
 
-					if (total > 1 || waiting)
+					if (first.Infinite)
+						/// Only regular has infinite symbol!
+						Game.Renderer.Fonts["Regular"].DrawTextWithContrast("∞",
```

Comment:
Can you please define an `infiniteFont` and assign its value from the ctor  (and move `overlayFont` there too)?